### ISSUE TITLE
weird bug

### ISSUE THREAD

**Issue by Heroburns** [TeamMidnightDust/BetterBeds#3]
idk if its mod i have but for some reason i cant see my chest?

**Comment by Kosta22024** [comment]
Yeah same for me

**Comment by haronv99** [comment]
comecei a ter também

**Comment by Theutus** [comment]
This happens to me as well, but only when using a resource pack. If I disable all resource packs, beds show up but if I have a resource pack active and this mod, I can't see any beds.

**Comment by Motschen** [comment]
Likely one of your resourcepacks overrides the bed model for some reason, try to put fabric resources on top of the list.

**Comment by PanoptesDreams** [comment]
Yes this happens to me a lot as well. Sometimes there, sometimes not.
Dev needs to fix.

**Comment by SinedKadis** [comment]
if i use quilt what should i do? I don't have mod resources in the resource packs folder
